coins.json

```diff
@@ -1260,5 +1260,26 @@
             "clientPublic": "", 
             "clientDocs": "https://polkadot.js.org/api/substrate/rpc.html"
         }
+    },
+    {
+        "id": "filecoin",
+        "name": "Filecoin",
+        "symbol": "FIL",
+        "decimals": 8,
+        "blockchain": "Filecoin",
+        "derivationPath": "m/44'/12345'/0'/0'",
```

Comment:
12345 should be replaced with real slip-44 ID of Filecoin, currently it is missing in https://github.com/satoshilabs/slips/blob/master/slip-0044.md

Reply:
See https://github.com/trustwallet/wallet-core/issues/780, this PR is not ready for review yet.

Reply:
Using SLIP-44 `1` now, the recommended value for any testnets

Reply:
Using SLIP-44 `1` for now, the recommended value for any testnet